tests/dualtor/test_tunnel_memory_leak.py

```diff
@@ -103,8 +103,27 @@ def check_memory_leak(duthost):
         return True
     return False
 
-def test_tunnel_memory_leak(
-    toggle_all_simulator_ports_to_upper_tor,
+def delete_neighbor(duthost, neighbor):
```

Comment:
should we move delete funciton to the dual_tor_utils.py? Below code is same as get_neighbor, it would be good to keep same function code in one place.
    command = "ip neighbor show %s" % neighbor
    output = [_.strip() for _ in duthost.shell(command)["stdout_lines"]]
    if not output:
        return {}
    output = output[0]
    neighbor_details = dict(_.split() for _ in itertools.chain(*re.findall('(dev\s+[\w\.]+)|(lladdr\s+[\w\.:]+)', output)) if _)

Reply:
@StormLiangMS thanks, it's a good suggestion, I just updated code, please check.